src/python/pants/backend/python/dependency_inference/rules.py

```diff
@@ -68,9 +68,9 @@ def register_options(cls, register):
             default=False,
             type=bool,
             help=(
-                "Infer a target's dependencies on any __init__.py files existing for the packages "
+                "Infer a target's dependencies on any `__init__.py` files in the packages "
```

Comment:
This is totally unrelated to the rest of the PR, but I happened to notice it while working on this PR, and it didn't justify a separate PR. Without the backticks the underscores are rendered as markdown, so this renders as <b>init<b>.py ...